app/assets/stylesheets/application.scss

```diff
@@ -37,4 +38,5 @@
 @import '@fullcalendar/list/main';
 @import "select2/dist/css/select2";
 @import "select2-bootstrap4-theme/dist/select2-bootstrap4.min";
-@import "jquery-datetimepicker/build/jquery.datetimepicker.min";
```

Comment:
@jjf21 could you double check that your IDE does add newlines at the end of files to avoid these kind of diffs?